frontend/src/components/reports/CancelSurveyResponseDialog.vue

```diff
@@ -1,15 +1,15 @@
 <template>
-  <AppDialog v-model="isDisplayed" title="Confirm" :isLoading="isLoading" persistent max-width="50%" @close="closeDialog">
+  <AppDialog v-model="isDisplayed" title="Confirm" :isLoading="loading" persistent max-width="50%" @close="closeDialog">
     <template #content>
       <div class="confirm-dialog-text d-flex flex-column align-center">Are you sure you want to cancel this report response?</div>
```

Comment:
This might be handled with the Delete story but we might want to update the text here. Not sure report response makes sense to the user.

Reply:
Agreed! we'll need to update the text of this box when we work on the delete ticket OFMCC-4126. I think we can use "restart your report", but I would like to get confirmation from Danna and the business.